server/db/db.js

```diff
@@ -4,123 +4,190 @@
 
 'use strict';
 
-var createHash = require('crypto').createHash;
-var log = require('../logger')('server.db.db');
-var utils = require('./utils');
+var crypto = require('crypto');
 
-var pool = utils.createPool();
+var pg = require('pg');
+var uuid = require('uuid');
```

Comment:
that actually was a really bad goof; visitId is optional, and the code should insert a `uuid.v4()` if the arg is missing